client/my-sites/plugins/constants.js

```diff
@@ -24,3 +24,5 @@ export const PREINSTALLED_PREMIUM_PLUGINS = {
 		},
 	},
 };
+
+export const AUTOMOMANAGED_PLUGINS_ECOM = [ 'woocommerce' ];
```

Comment:
Are you sure about the constant `AUTOMOMANAGED_PLUGINS_ECOM`? I feel there is a typo.

Reply:
changed to `AUTOMOMANAGED_PLUGINS` in [825a090](https://github.com/Automattic/wp-calypso/pull/67462/commits/825a090e53faa6ef13af449e99c020afbdc36876)